gno.land/cmd/gnoland/testdata/realm-banker-issued-coin-denom.txtar

```diff
@@ -0,0 +1,79 @@
+# test for https://github.com/gnolang/gno/pull/875
+
+## start a new node
+gnoland start
+
+## add realm_banker
+gnokey maketx addpkg -pkgdir $WORK/short -pkgpath gno.land/r/test/realm_banker -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+
+## add realm_banker with long package_name
+gnokey maketx addpkg -pkgdir $WORK/long -pkgpath gno.land/r/test/package89_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_1234567890 -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+
+## check initial balance
+gnokey query bank/balances/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5
+stdout '"9999996000000ugnot"'
+
+## mint coin from banker
+gnokey maketx call -pkgpath gno.land/r/test/realm_banker -func Mint -args "g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5" -args "ugnot" -args "31337" -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
```

Comment:
For clarity's sake, maybe try minting to an address that does not have any GNOTs in the beginning?

Reply:
`gnokey maketx send` all the money to the void :smile:

Reply:
0d39abdc2475d9e630961a5fd92b5dba0f0fb606